docs/cpp/config-clang-mac.md

```diff
@@ -131,155 +150,189 @@ Next, we need to create a `tasks.json` file to tell VS Code how to build (compil
+This task tells the C++ compiler to compile the active file (`${file}`), and create an output file (`-o` switch) in the current directory (`${fileDirname}`) with the same name as the active file (`${fileBasenameNoExtension}`), resulting in `helloworld.out` for our example.
 
-## Configure debug settings
+The `label` value is what you will see in the tasks list. Name this whatever you like.
 
-Next, we'll configure VS Code to launch the LLDB debugger when you press `kb(workbench.action.debug.start)`.
+The `problemMatcher` value selects the output parser to use for finding errors and warnings in the compiler output. For clang++.exe, you'll get the best results if you use the `$gcc` problem matcher.
+
+The `"isDefault": true` value in the `group` object specifies that this task will be run when you press `kb(workbench.action.tasks.build)`. This property is for convenience only; if you set it to `false`, you can still build from the Terminal menu with **Terminal > Run Build Task**.
+
+>**Note**: You can learn more about `task.json` variables in the [variables reference](/docs/editor/variables-reference.md).
+
+### Running the build
+
+1. Go back to `helloworld.cpp`. Because we want to build `helloworld.cpp` it is important that this file be the one that is active in the editor for the next step.
+1. To run the build task that you defined in tasks.json, press `kb(workbench.action.tasks.build)` or from the **Terminal** main menu choose **Run Build Task**.
+1. When the task starts, you should see the Integrated Terminal window appear below the code editor. After the task completes, the terminal shows output from the compiler that indicates whether the build succeeded or failed. For a successful Clang build, the output looks something like this:
+
+   ![Clang build output in terminal](images/clang-mac/clang-task-in-terminal.png)
 
-1. From the Command Palette, type "launch" and then choose **Debug: Open launch.json**. Next, choose the **GDB/LLDB** environment.
+1. Create a new terminal using the **+** button and you'll have a new terminal with the `helloworld` folder as the working directory. Run `ls` and you should now see the executable `helloworld` along with the debugging file (`helloworld.out.dSYM`).
 
-1. For `program`, use the program name `${workspaceFolder}/helloworld.out` (which matches what you specified in `tasks.json`).
+    ![Hello World in macOS terminal](images/clang-mac/helloworld-in-terminal.png)
 
-1. By default, the C++ extension adds a breakpoint to the first line of `main`. The `stopAtEntry` value is set to `true` to cause the debugger to stop on that breakpoint. You can set this to `false` if you prefer to ignore it.
+1. You can run `helloworld` in the terminal by typing `.\helloworld`.
```

Comment:
Should this be `./helloworld`, with a forward slash?

Reply:
Thank you for catching that. Fixed.